utils/cdk-runner/pkg/aws/cloudformation/stack.go

```diff
@@ -36,7 +36,7 @@ func GetStack(c *Client, stackName string) (*CfnStack, error) {
 		return cStack, err
 	}
 
-	if len(s.Stacks) > 0 {
+	if s != nil && len(s.Stacks) > 0 {
```

Comment:
We ignore the error generated above unless it's specifically the stack not existing. That means s can be nil here and cause a panic.